transfocate/tests/test_import.py

```diff
@@ -1,2 +1,2 @@
 def test_import_automated_checkout():
-    import transfocate.automated_checkout
+    pass
```

Comment:
What happened in this file? Is it worth keeping a test that's a no-op?

Reply:
Good catch - I think so.

pycln removed the unused import dutifully for us, but we actually want it here.